vmm/src/api/openapi/cloud-hypervisor.yaml

```diff
@@ -417,6 +417,9 @@ components:
         iommu:
           type: boolean
           default: false
+        seccomp:
+          type: boolean
+          default: true
```

Comment:
I don't think this should be here. The secomp control is part of the VMM control rather than for the VM.

Reply:
I agree, but I can also see why @likebreath took this approach. We don't really have a way to tell the VMM if we want seccomp to be enabled/disabled through the OpenAPI.